package.json

```diff
@@ -113,14 +114,17 @@
     "node-sass": "^3.2.0",
     "react-a11y": "0.2.6",
     "react-hot-loader": "1.3.0",
+    "react-transform-webpack-hmr": "^0.1.3",
     "redux-devtools": "^2.0.0",
     "sass-loader": "^2.0.0",
     "strip-loader": "^0.1.0",
     "style-loader": "^0.12.3",
     "webpack": "^1.9.11",
-    "webpack-dev-server": "1.10.1"
+    "webpack-dev-middleware": "^1.2.0",
+    "webpack-dev-server": "1.10.1",
+    "webpack-hot-middleware": "^2.0.0"
   },
   "engines": {
-    "node": ">=0.10.32"
+    "node": ">=2.2.1"
```

Comment:
Hmm, I thought one of the new dependencies required this version :/ My bad, I've force pushed a fix.